workflows/toast_sim_ground.py

```diff
@@ -591,6 +597,7 @@ def main():
         toast.ops.TimeConstant(
             name="convolve_time_constant", deconvolve=False, enabled=False
         ),
+        toast.ops.SaveHDF5(name="save_hdf5", enabled=False),
```

Comment:
We should also add the option to load data from file but that can wait for now.

Reply:
The `toast_sim_ground.py` workflow is set up primarily to do simulations from scratch starting with a scanning schedule and focalplane information.  In the case of using the `LoadHDF5` operator, we would be using other techniques to select a set of files to load.  All of the instrument information, boresight pointing, etc would already be provided, including detector data with some kinds of signal (real data or all simulated components or a subset of components, etc).  In that case I think it would make sense to have a different workflow with mostly data reduction operations.